src/core/Akka/IO/Inet.cs

```diff
@@ -106,7 +106,7 @@ public override void AfterConnect(Socket s)
                 }
             }
 
-            public class TrafficClass : SocketOption
+            /*TODO: this class is not used*/public class TrafficClass : SocketOption
```

Comment:
There is an outstanding TODO in that class to investigate .NET equivalent and port.